### PR TITLE
[Feature][Calling] Lock orientation

### PR DESCRIPTION
**Pull request by ShaunaSong**
## Purpose

* ...

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Pull request checklist

This PR has considered:
- [ ] Color Theming
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] View Data Injection
- [ ] Demo App UI/UX update

## How to Test

* Open...
* Click on...

## What to Check

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### CODE REVIEW

File: AzureCommunicationUI/Podfile.lock [hunk 1]
```diff
@@ -81,6 +82,6 @@ SPEC CHECKSUMS:
   SwiftLint: 4fa9579c63416865179bc416f0a92d55f009600d
   Trouter: 7eab36954ad740e32db3ff9f6025b0dff92193d7
 
-PODFILE CHECKSUM: 582b1bf1b10b1a2141a8c5b964d49b493bc14bf9
+PODFILE CHECKSUM: 2066d0cfc71e9457780a11d87feaeddfbdcfd32b
```
Comment (Mohtasim) [inline AzureCommunicationUI/Podfile.lock:85]: Is the podfile changes necessary for this PR?

Comment (ShaunaSong) [reply AzureCommunicationUI/Podfile.lock:85]: yes I add AzureCore dependency

Comment (Mohtasim) [reply AzureCommunicationUI/Podfile.lock:85]: I thought podfile.lock is generated locally and we do not push .lock files in PR. But I might be wrong here.

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/CallCompositeOptions/OrientationOptions.swift [hunk 2]
```diff
@@ -0,0 +1,74 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AzureCore
```
Comment (Mohtasim) [inline AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/CallCompositeOptions/OrientationOptions.swift:6]: AzureCore dependency has a problem with Xamarin build. Is AzureCore a must have for this Options file? Is there a way we can avoid AzureCore dependency?

Comment (ShaunaSong) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/CallCompositeOptions/OrientationOptions.swift:6]: Yes must have, if we want to follow Azure Review Board best practice for Enum. 

Also it is already included in ChatSDK as well.

**Review by Mohtasim — APPROVED** [review]

**Comment by ShaunaSong** [comment]
Closed. Have the other PR point to release

**Pull request closed without merging**